### PR TITLE
Add Tutanota exception

### PR DESCRIPTION
**Pull request by leggomyinfo**
Tutanota doesn't currently support U2F through their desktop or mobile apps. It is only supported when logging in through a browser. [According to Tutanota,](https://www.reddit.com/r/tutanota/comments/e5dti7/u2f_on_linux_app_security_key_not_registered_for/) support for this is coming "soon" as of a month ago. Since it's on their roadmap, I included "yet" in the exception language to differentiate them from companies who have not indicated eventual support.

Closes out #4515.

### CODE REVIEW

File: _data/email.yml
Comment (Carlgo11) [inline _data/email.yml]: Let's fix the hardware tag as well while we're at it.
```suggestion
      - u2f
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/email.yml
Comment (Carlgo11) [reply _data/email.yml]: firefox_PFbhu3qndV
There should be a "Commit suggestion" button available to you. Sadly I can't use it because of a GitHub bug 🙄

Comment (leggomyinfo) [reply _data/email.yml]: Should it have a leading space in front of the dash like " - hardware" does? By clicking "Commit changes," I am approving your additional change, right?

Comment (Carlgo11) [reply _data/email.yml]: The leading space is another GitHub bug hehehe 🙁  (It's not really there)
Yes the "Commit changes" incorporates my suggestion into the PR.

**Review by Carlgo11 — APPROVED** [review]
Great! Your first pull request 🥳 🎈 

Once another maintainer also approves it, your change will go live.

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**
